org.openhab.binding.zigbee/README.md

```diff
@@ -43,16 +55,52 @@ Once a child is removed from the child table of a router, it will be asked to re
 
 Note that ZigBee compliant devices should rejoin the network seamlessly, however some non-compliant devices may not rejoin which may leave them unusable without a manual rejoin.
 
+**Values:** Timeout time in seconds. The table below lists the options that are shown in PaperUI and the equivalent values that can be set in a configuration file:
+
+| Paper UI     | Config file |
+| ------------ | ------------|
+| _5 Minutes_  | `320`       |
+| _30 Minutes_ | `1800`      |
+| _2 Hours_    | `7200`      |
+| _12 Hours_   | `43200`     |
+| _1 Day_      | `86400`     |
+| _2 Days_     | `172800`    |
+| _5 Days_     | `432000`    |
+| _10 Days_    | `864000`    |
+| _2 Weeks_    | `1209600`   |
+| _4 Weeks_    | `2419200`   |
+| _7 Weeks_    | `4233600`   |
+
+Note that this value should be given as a number in the configuration file, without quotes
+
 ##### Concentrator Type (zigbee_concentrator)
 
 The Concentrator is used to improve routing within a ZigBee network, and is especially useful in a network where much of the traffic is sent to or from a central coordinator. If the coordinator has sufficient memory, it can store routing information, thus reducing network traffic.
 
 If supported, the High RAM concentrator should be used.
 
+**Values:**
+
+- `0` = Low RAM concentrator
+- `1` = High RAM concentrator
+
+
 ##### Mesh Update Period (zigbee_meshupdateperiod)
 
 The binding is able to search the network to get a list of what devices can communicate with other devices. This is a useful diagnostic feature as it allows users to see the links between devices, and the quality of these links. However, this can generate considerable traffic, and some battery devices may not poll their parents often enough to provide these updates, and users may consider that it is better to reduce the period, or disable this feature.
 
+**Value:** the update period in seconds. ```0``` means "never update". In PaperUI, a drop down list is shown, the options from that list are shown in te table below, with their equivalent values that can be put in a config file.
```

Comment:
I guess you could simply state that this is notionally defined in seconds rather than defining how many seconds are in each period?